docs/commands/heap.md

```diff
@@ -35,12 +38,15 @@ provide the address to the user memory pointer of the chunk to show the
 information related to a specific chunk:
 
 ```
-gef➤ heap chunk <LOCATION>
+gef➤ heap chunk [LOCATION]
 ```
 
 ![heap-chunk](https://i.imgur.com/SAWNptW.png)
 
-
+Because usually the heap chunks are aligned to a certain number of bytes in
```

Comment:
Actually they never need to add this option if the only look at the executable's heap (because it works for alll glibc versions). The option only needs to be added if they create fake heap chunks somewhere in memory that is not aligned and want to look at those fake heap chunks with GEF.

That's why I was also thinking of removing that option again and instead automatically enable it in the backend every time an address was provided by the user.